api/TextMetrics.json

```diff
@@ -101,7 +101,587 @@
           }
         }
       },
-      "advanced_text_metrics": {
+      "actualBoundingBoxLeft": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": true,
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Experimental Web Platform Features"
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": null
```

Comment:
Android should match chrome. This applies all the way through. Here's what Chrome has implemenbted: https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/html/canvas/text_metrics.idl

Reply:
Just pushed a commit that sets Chrome Android to match Chrome, and Webview to `false`.  How's it looking?